fontforge/splinefont_enums.h

```diff
@@ -64,6 +64,7 @@ enum ttf_flags {
     ttf_flag_dummyDSIG         = 1 << 15,
     ttf_native_kern            = 1 << 16, // This applies mostly to U. F. O. right now.
     ttf_flag_fake_map          = 1 << 17, // Set fake unicode mappings for unmapped glyphs
+    ttf_flag_no_outlines       = 1 << 18, // HarfBuzz: Skip "glyf" and "CFF " tables for performance
```

Comment:
@iorsh Hi! Here you're adding `ttf_flag_no_outlines` which is accidentally equal to `ps_flag_nohints`. So, after this change, a `font.generate()` call from Python with a `('no-hints')` flags passed generates a TTF without outlines, which is quite annoying and took some time to bisect and debug.
Probably `ttf_native_kern` (which is quite old) and `ttf_flag_fake_map` (which belongs to this changeset) which are conflicting with `ps_flag_nohintsubs` and `ps_flag_noflex`, also need to be addressed. And we're running out of bit space, so these values can't be just moved after 1 << 30. Probably `ps_flag_nohints` does not mean much in TTF building context, I am not sure; but changing the existing behaviour so that a corrupted file is generated seems to me a thing that should be avoided.